cli/src/cmd_net.rs

```diff
@@ -216,6 +218,12 @@ enum BgpSubCommand {
 
     /// Set a filtering specification for a peer.
     Filter(CmdBgpFilter),
+
+    /// Set an authentication string.
```

Comment:
It's not clear to me why this help string and the one on line 225 do not appear in the output.
```
$ oxide system networking bgp --help
Usage: oxide system networking bgp [OPTIONS] <COMMAND>

Commands:
  announce      Announce a prefix over BGP.
  announce-set  
  auth          
  config        
  exported      Get BGP exported routes
  filter        Add a filtering requirement to a BGP session.
  history       Get BGP router message history
  imported      
  peer          Manage BGP peers.
  pref          
  show-status   Get the status of BGP on the rack.
  status        Get BGP peer status
  withdraw      Withdraw a prefix over BGP.
  help          Print this message or the help of the given subcommand(s)

Options:
      --profile <PROFILE>  Configuration profile to use for commands
  -h, --help               Print help
```

It also doesn't appear here:
```
$ oxide system networking bgp auth --help

Usage: oxide system networking bgp auth [OPTIONS] --rack <RACK> --switch <SWITCH> --port <PORT> --peer <PEER>

Options:
      --rack <RACK>              Id of the rack to add the auth config to
      --profile <PROFILE>        Configuration profile to use for commands
      --switch <SWITCH>          Switch to add the auth config to [possible values: switch0, switch1]
      --port <PORT>              Port to add the auth config to [possible values: qsfp0, qsfp1, qsfp2, qsfp3, qsfp4, qsfp5, qsfp6, qsfp7, qsfp8, qsfp9, qsfp10, qsfp11, qsfp12, qsfp13, qsfp14, qsfp15, qsfp16, qsfp17, qsfp18, qsfp19, qsfp20, qsfp21, qsfp22, qsfp23, qsfp24, qsfp25, qsfp26, qsfp27, qsfp28, qsfp29, qsfp30, qsfp31]
      --peer <PEER>              Peer to add the auth config to
      --authstring <AUTHSTRING>  Use the given authorization string for TCP-MD5 authentication with the peer
  -h, --help                     Print help
```

Reply:
This is because those top-level structures are never included in `cli/src/main.rs`. See the `add_custom` directives in `main.rs` to see how commands are included in the overall CLI program. Because some top-level names like `bgp` must share their command namespace with commands that are autogenerated from the API, it is not possible to include them using `cmd_net`, so we need to cherry-pick more specific commands below the `bgp` part of the command.